### PR TITLE
v3.0 A3AA Update

### LINKED ISSUE #33: A3AA Update
Our Github repo needs to be updated to reflect the current state of @cnto_additions due to the A3AA Update by Freghar.

https://www.carpenoctem.co/forums/m/26081621/viewthread/16086660-a3sync-repository-status/post/139110263#p139110263

### PR DESCRIPTION
**Pull request by JamesTheClarke**
Updates our @cnto_additions to the latest version of Freghars A3AA.

Removed from Freghars original mod:
- Dynamic AI Skill
- misc_tiny_configs/sight_alignment

Added to Freghars mod:
- misc_tiny_configs/cnto_editor_subcategories

### CODE REVIEW

**Comment by JamesTheClarke** [comment]
@Seb105 when you got some time please review this pull request. Then I can publish the v3.0 release. :)

File: .gitignore [hunk 1]
```diff
@@ -1,30 +0,0 @@
-# arma-specific
```
Comment (milivojm) [inline .gitignore]: This file is useful to prevent junk files entering git repository.

File: addons/a3aa_functions/saveload/config.cpp [hunk 2]
```diff
@@ -1,19 +0,0 @@
-class CfgPatches {
```
Comment (milivojm) [inline addons/a3aa_functions/saveload/config.cpp:1]: saveload functions are useful for persistent missions and storing/restoring crate contents.

**Review by milivojm — APPROVED** [review]
I've returned Linux build scripts (useful) and .gitattributes, .gitignore. Everything else is battle tested for few months now.

**Pull request merged**